dspace/config/local.cfg.EXAMPLE

```diff
@@ -199,3 +199,9 @@ db.schema = public
 # Authentication by Password (encrypted in DSpace's database). See authentication-password.cfg for default configuration.
 # Enabled by default in authentication.cfg
 #plugin.sequence.org.dspace.authenticate.AuthenticationMethod = org.dspace.authenticate.PasswordAuthentication
+
+#### OpenSearch Settings ####
+# NB: for result data formatting, OpenSearch uses Syndication Feed Settings
+# so even if Syndication Feeds are not enabled, they must be configured
+# enable open search
+websvc.opensearch.enable = true
```

Comment:
Please remove this from the `local.cfg.EXAMPLE`.  Instead, it should be a comment in the main configuration `dspace.cfg` (as that's where the default settings for `websvc.opensearch.*` are).  

We are purposefully keeping the `local.cfg.EXAMPLE` very small (i.e. it only includes required settings or settings that are often changed).  As this isn't a required setting, it should be kept in the `dspace.cfg` file and described in Documentation for anyone who wants OpenSearch.

Reply:
done